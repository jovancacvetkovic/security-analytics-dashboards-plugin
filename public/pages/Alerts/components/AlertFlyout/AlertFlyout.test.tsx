/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import React from 'react';
import { render } from '@testing-library/react';
import { AlertFlyout } from './AlertFlyout';
import alertFlyoutMock from '../../../../../test/mocks/Alerts/components/AlertFlyout/AlertFlyout.mock';

describe('<AlertFlyout /> spec', () => {
  it('renders the component', () => {
    const { container } = render(<AlertFlyout {...alertFlyoutMock} />);
    expect(container).toMatchSnapshot();
  });
});
