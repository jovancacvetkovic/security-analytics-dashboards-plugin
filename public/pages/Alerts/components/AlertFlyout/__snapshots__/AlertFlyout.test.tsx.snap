// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertFlyout /> spec renders the component 1`] = `
<div
  aria-hidden="true"
  data-aria-hidden="true"
/>
`;
